includes/CLI/Plugin_Check_Command.php

```diff
@@ -232,25 +235,155 @@ static function ( $dirs ) use ( $excluded_directories ) {
 		}
 	}
 
+	/**
+	 * Runs plugin list-checks.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--fields=<fields>]
+	 * : Limit displayed results to a subset of fields provided.
+	 *
+	 * [--format=<format>]
+	 * : Format to display the results. Options are table, csv, and json. The default will be a table.
+	 * ---
+	 * default: table
+	 * options:
+	 *   - table
+	 *   - csv
+	 *   - json
+	 * ---
+	 *
+	 * @subcommand list-checks
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $args       List of the positional arguments.
+	 * @param array $assoc_args List of the associative arguments.
+	 *
+	 * @throws WP_CLI\ExitException Show error if not valid runner.
+	 */
+	public function list_checks( $args, $assoc_args ) {
+		$check_repo = new Default_Check_Repository();
+
+		$all_checks = array();
+
+		$checks_items = $check_repo->get_checks()->to_map();
+
+		foreach ( $checks_items as $key => $check ) {
```

Comment:
Could be simplified to:
```suggestion
		foreach ( $check_repo->get_checks() as $key => $check ) {
```
You don't need to call `to_map()`, since the object can be iterated as it implements `IteratorAggregate`.